### ISSUE TITLE
Wildly over-optimistic deltas

### ISSUE THREAD

**Issue by zenonp** [neo4j/neo4j#13274]
This is a sample output of a CSV import with neo4j-admin database import full: 

```
(3/4) Relationship linking 2023-08-04 01:30:55.705+0300
  Estimated required memory usage: 33.19GiB
.......... .......... .......... .......... ..........   5% ∆4m 42s 214ms
.......... .......... .......... .......... ..........  10% ∆4m 31s 875ms
.......... .......... .......... .......... ..........  15% ∆4m 29s 155ms
.......... .......... .......... .......... .........-  20% ∆201ms
.......... .......... .......... .......... ..........  25% ∆4m 11s 892ms
.......... .......... .......... .......... ..........  30% ∆7m 18s 328ms
.......... .......... .......... .......... ..........  35% ∆6m 44s 65ms
.......... .......... .......... .......... ..........  40% ∆7m 59s 17ms
.......... .......... .......... .......... ..........  45% ∆7m 48s 455ms
.......... .......... .......... .......... ..........  50% ∆7m 28s 246ms
.......... .......... .-........ .......... ..........  55% ∆6m 21s 785ms
.......... .......... .......... .......... ..........  60% ∆13m 9s 546ms
.......... .......... .......... .......... ..........  65% ∆16m 3s 260ms
.......... .......... .......... .......... ..........  70% ∆11m 42s 135ms
.......... .......... .......... .......... ..........  75% ∆11m 36s 827ms
.......... .......... .......... .......... ..........  80% ∆12m 40s 817ms
.......... .......... .......... .......... ..-.......  85% ∆8m 29s 489ms
........-. .......... .....-.... .......... ......-...  90% ∆22s 650ms
.......... .......-.. .......... ..-....... ......-...  95% ∆14m 36s 991ms
.......... ....-..... .......... .-........ -......... 100% ∆45m 40s 439ms

Relationship linking COMPLETED in 8h 15m 9s 85ms
```

The last line is correct; the process actually took 8h 15m. The deltas, however, are wildly off, as evidenced by the fact that their sum is only 3h 55m 48s. Seen on v5.10 on RHEL9. 

As an aside, printing milliseconds in this context seems rather redundant and only clutters the output.

**Comment by emmaholmbergohlsson** [comment]
Hey @zenonp 
Thank you for reporting. I had a look and found a bug around how those deltas were calculated. The fix will be included in the upcoming 5.13 release

**Comment by emmaholmbergohlsson** [comment]
Closing this issue since 5.13, which contains the fix, has been released. Feel free to reopen or open a new issue if you find that it still doesn't work for you with that fix. Thanks for reporting it @zenonp .
